identity/handler.go

```diff
@@ -175,11 +175,12 @@ type listIdentitiesParameters struct {
 //	Responses:
 //	  200: listIdentities
 //	  default: errorGeneric
-func (h *Handler) list(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+func (h *Handler) list(w http.ResponseWriter, r *http.Request, p httprouter.Params) {
 	var (
 		err    error
 		params = ListIdentityParameters{
 			Expand:                       ExpandDefault,
+			IdsFilter:                    r.URL.Query()["ids"],
```

Comment:
```suggestion
			IdsFilter:                    r.URL.Query().Get("ids"),
```

Reply:
.Get("ids") will only get first id and we need to populate slice here